app/models/admin_only/member_app_waiting_reason.rb

```diff
@@ -0,0 +1,25 @@
+module AdminOnly
+  
+  class MemberAppWaitingReason < ApplicationRecord
+
+    validates_presence_of :name_sv
```

Comment:
There is the whole area of being able to delete a `reason` that is actually in use (that is, one or more membership_applications have a `has_one` association with the `reason` instance).

We could add a validation that invalidates a `destroy` or `delete` action if the `reason` has one or more associated MA's.  This might be fine-tuned to allow destroy if none of the associated MA's are in the "waiting for information" state.  (on the other hand, we probably want to retain the `reason` anyway - for reporting and audit reasons).

Also, the "Delete" button on the edit and update views should be missing (or greyed out and not active) if the above situation holds.